web/sites/default/settings.lando.php

```diff
@@ -0,0 +1,33 @@
+<?php
+
+// @codingStandardsIgnoreFile
+
+/**
+ * @file
+ * Drupal site-specific configuration file.
+ */
+
+// Lando environment settings
+if (getenv('LANDO_INFO')) {
+  $lando_info = json_decode(getenv('LANDO_INFO'), TRUE);
+
+  // Files directory paths.
+  $settings['file_public_path'] = 'sites/default/files';
+  $settings['file_private_path'] = 'sites/default/files/private';
```

Comment:
Let's remove this section. Same as my previous comment: the use case for this repo is that different users get the same site whether they use Lando or DDev. The use case is _not_ that we run two different sites from the same repo, one using Lando and one using DDev.

Reply:
![Animated gif of a finger pressing "Delete" key repeatedly](https://user-images.githubusercontent.com/1103978/40451702-7990f956-5ead-11e8-9b7d-9ebfaec02d00.gif)